cmd/kubeadm/app/apis/bootstraptoken/v1/utils_test.go

```diff
@@ -648,7 +648,7 @@ func TestBootstrapTokenFromSecret(t *testing.T) {
 					Name:      rt.name,
 					Namespace: "kube-system",
 				},
-				Type: v1.SecretType("bootstrap.kubernetes.io/token"),
+				Type: "bootstrap.kubernetes.io/token",
```

Comment:
this is also present on line 278, can we import 
`bootstrapapi "k8s.io/cluster-bootstrap/token/api"`

and use `bootstrapapi.SecretTypeBootstrapToken`?

Reply:
Good idea! Fixed.